src/ape_console/plugin.py

```diff
@@ -0,0 +1,66 @@
+import click
+from click.testing import CliRunner
+from eth_utils import to_hex
+from IPython import get_ipython  # type: ignore
+from IPython.core.magic import Magics, line_magic, magics_class  # type: ignore
+
+import ape
+from ape._cli import cli
+
+
+@magics_class
+class ApeConsoleMagics(Magics):
+    @line_magic
+    def ape(self, line: str = ""):
+        """
+        Run Ape CLI commands within an ``ape console`` session.
+
+        Usage example::
+
+            %ape accounts list
+
+        """
+        runner = CliRunner()
```

Comment:
Lol, clever use of the test runner

Reply:
ya, is this ok, do you think?

Reply:
Seems fine to me, it's just a wrapper anyways I think